scripts/Dockerfile-testInstall

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:latest
```

Comment:
Nice!

Note that this doesn't test installing as a non-root user. Definitely better than nothing though!